ResourceStatusSystem/Agent/EmailAgent.py

```diff
@@ -0,0 +1,148 @@
+''' EmailAgent
+
+  This agent sends an email every 30 minutes with aggregated actions taken for each site
```

Comment:
I'd add some more details of what is actually done. At the same time, "30 minutes" is a default.